store/src/db/mod.rs

```diff
@@ -277,7 +294,7 @@ impl Db {
                     .get()
                     .await?
                     .interact(move |conn| -> anyhow::Result<()> {
-                        let span = span!(Level::INFO, COMPONENT, "writing genesis block to DB");
+                        let span = info_span!(target: COMPONENT, "setup:ensure_genesis_block:writing genesis block to DB");
```

Comment:
Similar comment to the ones before: let's not manually prefix the span name here and leave a `TODO`.

Also, let's change the span name to "write_genesis_block_to_db".